src/main/webapp/resources/js/pages/password/ActivateAccount.jsx

```diff
@@ -0,0 +1,104 @@
+import { useActivateAccountMutation } from "../../apis/passwordReset";
+import { Alert, Button, Form, Input } from "antd";
+import React, { useEffect, useRef } from "react";
+import { setBaseUrl } from "../../utilities/url-utilities";
+import { SPACE_MD } from "../../styles/spacing";
+const { Item } = Form;
+
+/**
+ * React component to render the forgot password form
+ * @param {function} updateDisplayLoginPage Function to update whether to display login page
+ * @returns {*}
+ * @constructor
+ */
+export function ActivateAccount({ updateDisplayLoginPage }) {
+  const [activateAccount] = useActivateAccountMutation();
+  const [activateAccountForm] = Form.useForm();
+  const [messageAlert, setMessageAlert] = React.useState(false);
+  const [message, setMessage] = React.useState("");
+  const activationIdRef = useRef();
+
+  /**
+   * When the component gets added to the page,
+   * focus on the activationId input.
+   */
+  useEffect(() => {
```

Comment:
For consistency use `React.useEffect`

Reply:
Updated in 66fd23c47ca7b634ae1918ac77341a2fdf53048e.